tests/assets/plugins/taggers/plugin_third_party_trainable_tagger.py

```diff
@@ -180,7 +180,7 @@ def model_cls(self) -> Type[ThirdPartyModel]:
     def run_with_model(
         self, request: PluginRequest[BlockAndTagPluginInput], model: TestTrainableTaggerModel
     ) -> InvocableResponse[BlockAndTagPluginOutput]:
-        """Downloads the model file from the provided space"""
+        """Downloads the model file from the provided workspace"""
```

Comment:
No need to change it in this PR but a heads-up. As soon as we enable docstring checking, this will get flagged for not ending with a period.